### PR TITLE
core(render-blocking-resources): add FCP and LCP savings

### PR DESCRIPTION
**Pull request by adrianaixba**

### CODE REVIEW

File: core/audits/byte-efficiency/render-blocking-resources.js [hunk 1]
```diff
@@ -299,6 +299,7 @@ class RenderBlockingResources extends Audit {
       numericValue: wastedMs,
       numericUnit: 'millisecond',
       details,
+      metricSavings: {FCP: wastedMs, LCP: wastedMs},
```
Comment (adamraine) [inline core/audits/byte-efficiency/render-blocking-resources.js:302]: Using the existing `wastedMs` makes sense for now, but `wastedMs` is computed using the FCP graph. I do wonder if computing a second LCP `wastedMs` would also be useful.

This requires more investigation, so I'll open a separate issue to track this.

**Review by adamraine — APPROVED** [review]

**Comment by adamraine** [comment]
Smoke test failures look unrelated :/

File: core/audits/byte-efficiency/render-blocking-resources.js (hunk 1 above)
Comment (adrianaixba) [reply core/audits/byte-efficiency/render-blocking-resources.js:302]: ah something like what we have to do with `uses-rel-preconnect`? 

separate issue sounds good to me!

Comment (adamraine) [reply core/audits/byte-efficiency/render-blocking-resources.js:302]: Yeah, I imagine we'll copy the FCP implementation for LCP and see how big the difference is.

**Pull request merged**